# NGS-Tools-Reference
A curated list of NGS tools mentioned in our chapter, including tool names and their official links. This repository serves as a quick reference for readers and researchers.

**General NGS Data Analysis**

FastQC – Quality control for high throughput sequence data
https://www.bioinformatics.babraham.ac.uk/projects/fastqc/

MultiQC – Aggregate QC reports from multiple tools
https://multiqc.info/

Trimmomatic – Read trimming and filtering
http://www.usadellab.org/cms/?page=trimmomatic

Cutadapt – Adapter trimming
https://cutadapt.readthedocs.io/en/stable/

Fastp – Fast and all-in-one preprocessor for FASTQ files
https://github.com/OpenGene/fastp

**Read Alignment**

BWA – Burrows-Wheeler Aligner
http://bio-bwa.sourceforge.net/

Bowtie2 – Ultrafast and memory-efficient tool for aligning sequencing reads
http://bowtie-bio.sourceforge.net/bowtie2/

STAR – Spliced Transcripts Alignment to a Reference (RNA-seq)
https://github.com/alexdobin/STAR

HISAT2 – Fast and sensitive alignment of NGS reads
https://daehwankimlab.github.io/hisat2/

**Post-alignment Processing**

SAMtools – Tools for manipulating alignments in SAM/BAM format
http://www.htslib.org/

Picard – A set of Java command-line tools for manipulating high-throughput sequencing data
https://broadinstitute.github.io/picard/

GATK – Genome Analysis Toolkit (variant discovery)
https://gatk.broadinstitute.org/

**Transcriptomics/RNA-seq**

HTSeq – Python framework for counting aligned reads
https://htseq.readthedocs.io/en/master/

featureCounts (Subread) – Assigns reads to genomic features
http://bioinf.wehi.edu.au/featureCounts/

DESeq2 – Differential expression analysis based on the negative binomial distribution
https://bioconductor.org/packages/release/bioc/html/DESeq2.html

edgeR – Empirical analysis of digital gene expression data
https://bioconductor.org/packages/release/bioc/html/edgeR.html

**CRISPR Screen Analysis**

MAGeCK – Model-based Analysis of Genome-wide CRISPR/Cas9 Knockout
https://sourceforge.net/p/mageck/wiki/Home/

CRISPResso – Analysis of genome editing outcomes
https://crispresso.pinellolab.partners.org/

PinAPL-Py – CRISPR pooled screen analysis pipeline
https://pinapl-py.readthedocs.io/en/latest/

JACKS – Joint Analysis of CRISPR/Cas9 Knockout Screens
https://github.com/felicityallen/JACKS

**ChIP-seq / ATAC-seq**

MACS2 – Model-based Analysis for ChIP-Seq
https://github.com/macs3-project/MACS

HOMER – Motif discovery and next-gen sequencing analysis
http://homer.ucsd.edu/homer/

